src/trace_processor/importers/etm/virtual_address_space.cc

```diff
@@ -49,17 +49,18 @@ void VirtualAddressSpace::Builder::AddMapping(
   AddressRange range(static_cast<uint64_t>(mapping.start()),
                      static_cast<uint64_t>(mapping.end()));
 
-  std::optional<TraceBlobView> content;
+  std::optional<TraceBlob> content;
   if (mmap.file_id()) {
-    content = context_->registered_file_tracker->GetContent(*mmap.file_id());
-    auto file_range = AddressRange::FromStartAndSize(0, content->size());
+    TraceBlob& blob =
+        context_->registered_file_tracker->GetContent(*mmap.file_id());
+    auto file_range = AddressRange::FromStartAndSize(0, blob.size());
     auto required_file_range = AddressRange::FromStartAndSize(
         static_cast<uint64_t>(mapping.exact_offset()), range.size());
 
     PERFETTO_CHECK(file_range.Contains(required_file_range));
 
-    content = content->slice_off(required_file_range.start(),
-                                 required_file_range.length());
+    content = TraceBlob::CopyFrom(blob.data() + required_file_range.start(),
```

Comment:
Please leave a comment that this is inefficient and we should switch back to using TraceBlobView.